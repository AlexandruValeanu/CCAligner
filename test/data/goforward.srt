1
00:00:00,500 --> 00:00:02,469
Go forward ten meters