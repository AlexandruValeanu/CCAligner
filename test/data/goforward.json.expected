{
	"subtitles": [
	{
		"subtitle" : "Go forward ten meters",
		"edited_text" : "Go forward ten meters",
		"start" : "500",
		"end" : "2469",
		"words" : [
			{
				"word" : "Go",
				"recognised" : "1",
				"start" : "530",
				"end" : "660",
				"duration" : "130"
			},
			{
				"word" : "forward",
				"recognised" : "1",
				"start" : "670",
				"end" : "1220",
				"duration" : "550"
			},
			{
				"word" : "ten",
				"recognised" : "1",
				"start" : "1230",
				"end" : "1540",
				"duration" : "310"
			},
			{
				"word" : "meters",
				"recognised" : "1",
				"start" : "1550",
				"end" : "2140",
				"duration" : "590"
			}
		],
		"phonemes" : [
		]
	}	]
}
